### PR TITLE
Maid-1059

### PR DESCRIPTION
**Pull request by hintofbasil**
https://maidsafe.atlassian.net/projects/MAID/issues/MAID-1059

rename types::Action -> types::MessageAction; rename RoutingNodeAction -> MethodCall

[Review on Reviewable](https://reviewable.io/reviews/maidsafe/routing/263)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
Thanks for the pull request, and welcome! The MaidSafe team is excited to review your changes, and you should hear from @benjaminbollen (or someone else) soon.

If any changes to this PR are deemed necessary, please add them as extra commits. This ensures that the reviewer can see what has changed since they last reviewed the code. The way Github handles out-of-date commits, this should also make it reasonably obvious what issues have or haven't been addressed. Large or tricky changes may require several passes of review and changes.

Please see [CONTRIBUTOR.md](../blob/master/CONTRIBUTOR) for more information.

**Comment by dirvine** [comment]
Just a quick than you note and welcome to the team/community, very much appreciated and another beer owed :-)

**Comment by benjaminbollen** [comment]
Yes, thank you very much !
I'll review it soon - sorry I was out for my birthday last night :)

File: README.md [hunk 1]
```diff
@@ -58,7 +58,6 @@ correct name calculation of pure Id
     - [ ] [MAID-1054](https://maidsafe.atlassian.net/browse/MAID-1054) move all handler functions to separate module
     - [ ] [MAID-1055](https://maidsafe.atlassian.net/browse/MAID-1055) full review of implementation of handlers
     - [ ] [MAID-1057](https://maidsafe.atlassian.net/browse/MAID-1057) make event loop in routing_node internal
-    - [ ] [MAID-1059](https://maidsafe.atlassian.net/browse/MAID-1059) rename types::Action -> types::MessageAction; rename RoutingNodeAction -> MethodCall
```
Comment (benjaminbollen) [inline README.md]: Thanks, but we will mark it as done, so I will take this change out; or you can do it.

**Comment by hintofbasil** [comment]
Glad I can help out.

I wasn't sure which was to go with marking a TODO as done.  Seems I guessed wrong, I've updated it now.  Hopefully that's all good to be merged.

Also happy birthday.

**Comment by benjaminbollen** [comment]
Thanks @hintofbasil for your patience.  As this work changes the api of routing to other libraries, I wanted to hold it off a bit until we bumped to a new version.  I can merge it now in v0.1.60.  Thanks!

**Pull request merged**